frontend/catalyst/device/decomposition.py

```diff
@@ -124,7 +124,18 @@ def _decompose_nested_tapes(op, ctx, stopping_condition, decomposer, max_expansi
             )
         )
 
-    new_op = op.__class__(op.in_classical_tracers, op.out_classical_tracers, new_regions)
+    # Copy the hybrid operation. QCtrl needs a special branch only due to its custom parameters.
+    if isinstance(op, QCtrl):
```

Comment:
Did this come up in testing somehow? My understanding was that `QCtrl` should never make it here because it will be decomposed by the call to the PennyLane `decompose function` before `_decompose_nsted_tapes` is called, since it's not a supported op but is decomposed in the frontend into PennyLane `Controlled` operations. 

It will need special handling later if we make it stop doing that, but it will need more than just recursing into the tapes.

Reply:
In tests I disable decomposition to test the verification use-cases.